hw/syn/tools/dc/run-syn.tcl

```diff
@@ -103,9 +109,19 @@ source constraints.sdc
 ##    MAP DESIGN    ##
 ######################
 
-# TODO: we may have to disable a couple of optimizations in order
-# to prevent the tool from optimizing away dummy logic or logic from blocks
-# that are only half-finished
+# disable constant propagation
```

Comment:
is this meant as a temporary TODO?

It feels like the design could bloat potentially if there are a lot of hardwired things (for example fabric IDs etc)

Reply:
yes this is only meant for experiments on bronze since there are several unconnected things that may potentially be optimized away. you are right that with all these optimizations disabled, the area is likely over estimated.

once everything is wired up we should look at the reports and area distribution and check whether some of these options can be turned on again.